stable/pvc/templates/_pvc.tpl

```diff
@@ -0,0 +1,19 @@
+{{- define "pvc.volumes" -}}
```

Comment:
should the `pvc` level be here?
I'm not sure, I just see `pvc: ...` is in jupyter and pvc helm chart has no values of it's own

Reply:
it's the value of pvc.pvcVolumesTemplate. 
It's the same as in v3io-configs